tools/cp-caps/rcp_caps_test.py

```diff
@@ -635,22 +645,26 @@ def parse_arguments():
 def main():
     arguments = parse_arguments()
 
-    if arguments.verbose is True:
+    if arguments.version:
+        print(f'Version: {CP_CAPABILITY_VERSION}')
+        exit()
+
+    if arguments.verbose:
         logger = logging.getLogger()
         logger.setLevel(logging.DEBUG)
 
     rcp_caps = RcpCaps()
 
-    if arguments.diag_commands is True:
+    if arguments.diag_commands:
         rcp_caps.test_diag_commands()
 
-    if arguments.csl is True:
+    if arguments.csl:
```

Comment:
perhaps we can switch to the `unittest` framework for better filtering tests in a separate PR.

Reply:
We can switch to unittest in other commit.